src/ucp/core/ucp_ep.c

```diff
@@ -632,24 +634,23 @@ void ucp_ep_config_init(ucp_worker_h worker, ucp_ep_config_t *config)
 
                 if (context->config.ext.zcopy_thresh == UCS_CONFIG_MEMUNITS_AUTO) {
                     /* auto */
-                    zcopy_thresh = md_attr->reg_cost.overhead / (
-                                            (1.0 / context->config.ext.bcopy_bw) -
-                                            (1.0 / iface_attr->bandwidth) -
-                                            md_attr->reg_cost.growth);
-                    if (zcopy_thresh < 0) {
-                        config->am.zcopy_thresh      = SIZE_MAX;
-                        config->am.sync_zcopy_thresh = -1;
-                    } else {
-                        config->am.zcopy_thresh      = zcopy_thresh;
-                        config->am.sync_zcopy_thresh = zcopy_thresh;
+                    config->am.zcopy_thresh[0]      = UCS_CONFIG_MEMUNITS_AUTO;
```

Comment:
i see. maybe better to keep it as separate flag then? this way is a bit confusing